src/profile/utilities.jl

```diff
@@ -0,0 +1,165 @@
+"""
+    TableColumns
+
+A structure containing the column names for the numeric part of the profile table.
+
+The struct also contains a Dict giving the column ranges for Symbols like `:σ` and `:β`.
+Finally it contains a scratch vector used to accumulate to values in a row of the profile table.
+"""
+struct TableColumns{T<:AbstractFloat}
+    cnames::Tuple
```

Comment:
It might make sense to have

```suggestion
    cnames::Vector{Symbol}
```

We're already allocating a vector for constructing this struct and then we have only concrete, non-union types as fields. That means the struct itself can be allocated more efficiently.